README.adoc

```diff
@@ -1,24 +1,30 @@
 == Prerequisites
 
-* a running 1.7 Kubernetes cluster with at least 4GB of memory
-these instructions assume minikube for now, for example:
+* A running 1.7 Kubernetes cluster with at least 4GB of memory
+these instructions assume minikube for now. Point your docker environment
+to the docker running in minikube.
 +
 ```
 minikube start --cpus=4 --memory=4096 --disk-size=64g
+eval $(minikube docker-env)
```

Comment:
can we remove line 36 if we have this here?

Reply:
i thought about removing it there, but since the docker-env needs to be set for every terminal session i thought it best to keep it in both places.